video/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>videointelligence-snippets</artifactId>
+  <packaging>jar</packaging>
+  <name>Google Cloud Video Intelligence Snippets</name>
+  <url>https://github.com/googleapis/java-video-intelligence</url>
```

Comment:
Can we update this to: `https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/video`

Reply:
good catch. fixed.